vaadin-spring/src/test/java/com/vaadin/flow/spring/SpringStubServletConfigTest.java

```diff
@@ -1,84 +0,0 @@
-package com.vaadin.flow.spring;
-
-import javax.servlet.ServletContext;
-
-import java.util.Collections;
-
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
-import org.mockito.Mock;
-import org.mockito.Mockito;
-import org.mockito.MockitoAnnotations;
-import org.springframework.boot.web.servlet.ServletRegistrationBean;
-import org.springframework.context.ApplicationContext;
-import org.springframework.core.env.Environment;
-
-import com.vaadin.flow.di.Lookup;
-import com.vaadin.flow.di.ResourceProvider;
-import com.vaadin.flow.function.DeploymentConfiguration;
-import com.vaadin.flow.server.Constants;
-
-import static org.mockito.Mockito.when;
-
-/**
- * Test for checking that styb servlet configuration collects expected values.
- */
-public class SpringStubServletConfigTest {
-
-    @Mock
-    private ApplicationContext applicationContext;
-
-    @Mock
-    private Environment environment;
-
-    @Mock
-    private ServletContext context;
-
-    @Mock
-    private ServletRegistrationBean registration;
-
-    @Before
-    public void init() {
-        MockitoAnnotations.initMocks(this);
-
-        Lookup lookup = Mockito.mock(Lookup.class);
-        ResourceProvider provider = Mockito.mock(ResourceProvider.class);
-        Mockito.when(lookup.lookup(ResourceProvider.class))
-                .thenReturn(provider);
-
-        Mockito.when(context.getAttribute(Lookup.class.getName()))
-                .thenReturn(lookup);
-
-        when(applicationContext.getBean(Environment.class))
-                .thenReturn(environment);
-        when(registration.getServletName()).thenReturn("vaadinServlet");
-        when(context.getInitParameterNames())
-                .thenReturn(Collections.emptyEnumeration());
-        when(registration.getInitParameters())
-                .thenReturn(Collections.emptyMap());
-
-    }
-
-    @Test
-    public void vaadinProductionMode_isReadFromEnvironmentVariables() {
```

Comment:
Non-blocking question: Would this have made any sense anymore as another test without the stub config ?

Reply:
If not, good riddance. If yes, then can be added back later (to get this in)

Reply:
There is already a test which replaces this one.
Except this one checks only one single property `productionMode`.
`SpringApplicationConfigurationFactoryTest`  tests all `SpringServlet.PROPERTY_NAMES` properties.
And `SpringApplicationConfigurationFactory` is the proper SPI impl replacement for ugly stub config